controllers/grid/settings/submissionChecklist/SubmissionChecklistGridHandler.inc.php

```diff
@@ -124,7 +124,7 @@ function editItem($args, $request) {
 		$submissionChecklistId = isset($args['rowId']) ? $args['rowId'] : null;
 		$submissionChecklistForm = new SubmissionChecklistForm($submissionChecklistId);
 
-		$submissionChecklistForm->initData($args, $request);
+		$submissionChecklistForm->initData($args);
```

Comment:
(As above -- not sure if `$args` is still needed)

Reply:
As above; `SubmissionChecklistForm` relies on a few things being passed in.